INSTALL.md

```diff
@@ -275,6 +276,84 @@ librehealthehr/sites/default/letter_templates
+
+2. php 7.1x isn't supported
+
+Clone the LibreEHR [repository](https://github.com/LibreHealthIO/LibreEHR) into your console eg. [GitBash](https://git-for-windows.github.io/), [Cmder](http://cmder.net/)
+
+After cloning LibreEHR, the `php.ini` file is located in `xampp\php\` directory.
+
+Open this file with your text editor eg. [Subl](https://www.sublimetext.com/3).
+
+There will be 4 php files located in `xampp\php`, but the one needed is the one that's type is "Configuration Settings" that  change the following in the php.ini file:
+
+```
+max_execution_time = 600
+max_input_time = 600
+max_input_vars = 3000
+memory_limit = 512M
+post_max_size = 32M
+upload_max_filesize = 32M
+session.gc_maxlifetime = 14400
+short_open_tag = On
+display_errors = Off
+upload_tmp_dir is set to a correct default value that will work on your system
+error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT & ~E_NOTICE
+```
+
+**Make sure that strict mode is disabled in Mysql**
+
+**[How to disable Mysql strict mode?](#how-to-disable-mysql-strict-mode-?)**
+
+#### Step 1
+Navigate to LibreEHR Setup:
+localhost/libreEHR/setup.php
+
+**Note:**
+1. Make sure that your XAMPP control panel has apache, mySQL, and Filezilla turned on
+2. Apache needs to be on port 80, 443
+3. mySQL needs to be on port 3306
+4. Filezilla needs to be on port 21, 14147
+
+#### Step 2
+Leave default as the "Site ID:" and press continue.
+
+![First Step](./Documentation/1_Installing/images/windows_installation/Step_1.png)
+
+Make sure that there are no undefined index errors, if so make sure that you changed the php.ini file, or have the correct version of XAMP.
+
+![Second Step](./Documentation/1_Installing/images/windows_installation/Step_2.png)
+
+Then after the second step, continue and leave the option "Have setup create the database" and press continue
+
+![Third Step](./Documentation/1_Installing/images/windows_installation/Step_3.png)
+
+For the fourth step, enter a "Password" and "Initial User Password" you are free to change, the "Initial User" to your own username, but for convenience you can also leave it as admin. Then press continue
+
+![Fourth Step](./Documentation/1_Installing/images/windows_installation/Step_4.png)
+
+Then after the fourth step, you can press continue through the others as long as the steps above were followed with clarity, the rest should have no errors, and each remaining page can be continued without change.
+
+![Fifth Step](./Documentation/1_Installing/images/windows_installation/Step_5.png)
+
+![Sixth Step](./Documentation/1_Installing/images/windows_installation/Step_6.png)
+
+![Seventh Step](./Documentation/1_Installing/images/windows_installation/Step_7.png)
+
+![Eigth Step](./Documentation/1_Installing/images/windows_installation/Step_8.png)
+
+![Ninth Step](./Documentation/1_Installing/images/windows_installation/Step_9.png)
+
+**Yay, you have successfull setup LibreEHR!**
+
```

Comment:
I agree with your banner on the screen shots but others might take offence. Can you mask it please?